### PR TITLE
Brazilian Portuguese translation of Gear up for Success.

### PR DESCRIPTION
**Pull request by christophermoura**

### CODE REVIEW

**Comment by bugron** [comment]
@Janainaf @lucaslmuller can you QA, please?

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 1]
```diff
@@ -100,7 +127,7 @@
       "tests": [],
```
Comment (panyarerkcap) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: [[50,000]]

**Comment by christophermoura** [comment]
@Janainaf @lucaslmuller could you QA, please?

**Comment by daianepiva** [comment]
Hi Christopher, this is the first time I'll comment on an open question, I hope I'm doing it in the right place; 
So, gear translates to "marcha" so, in Portuguese we have "primeira marcha ate quinta". It's like saying "quinta marcha em rumo ao sucesso", "marcha total em rumo ao sucesso", "acelere em rumo ao sucesso", "vai com tudo em rumo ao sucesso" or anything close to that. 

Hope it helps.

**Comment by christophermoura** [comment]
Hi @daianepiva!

First of all, thank you for the input!

Actually, the expression "Gear Up" means something like "_To step up to the challenge_". That way, the translation to portuguese would be "**Prepare-se para o sucesso**".

Also, could you point in the code where this translation is missing? If it's the title, I will question @vtamara about it because I haven't seen the title being translated in the spanish translation.

**Comment by bugron** [comment]
@christophermoura please, `rebase` your branch on top of the latest staging branch.

**Comment by BerkeleyTrue** [comment]
Closing as stale. Feel free to re-open when ready

**Pull request closed without merging**